components/product_notices_modal/product_notices_modal.tsx

```diff
@@ -186,6 +188,11 @@ export default class ProductNoticesModal extends React.PureComponent<Props, Stat
         return null;
     }
 
+    private trackClickEvent = () => {
+        const presentNoticeInfo = this.state.noticesData[this.state.presentNoticeIndex];
+        trackEvent('ui', `notice_click_${presentNoticeInfo.id}`);
```

Comment:
not to change it right now, but feels like `'ui'` should really be a constant so we use the same string throughout the code prevent typos

Reply:
Ya, i agree. All events strings need to be changed to constants. I think we can run a campaign on this